charts/k8s-service/values.yaml

```diff
@@ -496,6 +499,39 @@ horizontalPodAutoscaler:
   minReplicas: 1
   maxReplicas: 10
 
+# customObjects is a map that lets you define Kubernetes objects you want installed and configured as part of this chart.
+# The expected keys of customObjects are:
+#   - enabled     (bool) : Whether or not the provided definitions should be created.
```

Comment:
Why a separate `enabled` flag? Is it not enough to see if `customObjects` is empty or not?

Reply:
This is a general pattern in helm and is best practice due to the way merges happen and how helm sets things from the CLI. It is typically very hard to empty out an object if something already sets it in your pipeline, than it is to set an attribute.

Reply:
Ohhh, got it, thanks for the explanation.